base/src/PipeLine.cpp

```diff
@@ -215,6 +225,7 @@ void PipeLine::step()
 			i->get()->queueStep();
 		}
 	}
+	// should controlModule step ?
```

Comment:
I don't think it makes sense for control module to step

Reply:
command  processing also happens when a module steps - I am writing a test where control module steps to process a command